HouseRules_Essentials/Rules/PieceAbilityListOverriddenRule.cs

```diff
@@ -31,9 +31,10 @@ public PieceAbilityListOverriddenRule(Dictionary<BoardPieceId, List<AbilityKey>>
         protected override void OnPostGameCreated(GameContext gameContext)
         {
             var pieceConfigs = Resources.FindObjectsOfTypeAll<PieceConfig>();
+            string lookupstring = string.Empty;
```

Comment:
`lookupstring` looks unused in all of the files.

Reply:
Good spot - that was a left-over from before I moved the FixBossNames method to a common location.